packages/react/dropdown-menu/src/DropdownMenu.tsx

```diff
@@ -155,15 +155,13 @@ const DropdownMenuContent = forwardRefWithAs<
         // as it's already setup to close, otherwise it would close and immediately open.
         if (wasTrigger) {
           event.preventDefault();
-        } else {
```

Comment:
I also removed this as I don't think it makes sense, as that event is already called internally in all cases (focus outside, pointer down outside).